clarity/contracts/equations/weighted-equation.clar

```diff
@@ -14,6 +14,7 @@
 ;; max in/out as % of liquidity
```

Comment:
Below update on this file is about tolerance parameter on equation, I just commented it. I think I can just deprecate it or apply Chan's approach on yield-token-equation which returns u0 when the value is capped